src/extension/handler/uri.ts

```diff
@@ -60,9 +66,67 @@ export class RunmeUriHandler implements UriHandler {
       return
     }
 
+    if (command === 'demo') {
+      try {
+        const { fileToOpen, repository, cell } = parseParams(params)
+        if (!repository) {
+          throw new Error('Could not find a valid git repository')
+        }
+
+        if (!cell || Number.isNaN(cell)) {
```

Comment:
@degrammer with zero based-index this needs to be `cell === undefined` because `0` is falsey.